egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh

```diff
@@ -54,23 +54,25 @@ pruned_rescore=true
 . ./cmd.sh
 . ./utils/parse_options.sh
 
-# test of 960 hours training transcriptions
-text=data/train_960/text
+text=data/local/lm/librispeech-lm-norm.txt.gz
 lexicon=data/lang_nosp/words.txt
-text_dir=data/rnnlm/text_960_1a
+text_dir=data/rnnlm/text
 mkdir -p $dir/config
 set -e
 
-for f in $text $lexicon; do
+for f in $lexicon; do
   [ ! -f $f ] && \
     echo "$0: expected file $f to exist; search for run.sh in run.sh" && exit 1
 done
 
 if [ $stage -le 0 ]; then
   mkdir -p $text_dir
+  if [ ! -f $text ]; then
+    wget http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz -P data/local/lm 
+  fi
   echo -n >$text_dir/dev.txt
-  # hold out one in every 50 lines as dev data.
-  cat $text | cut -d ' ' -f2- | awk -v text_dir=$text_dir '{if(NR%50 == 0) { print >text_dir"/dev.txt"; } else {print;}}' >$text_dir/librispeech.txt
+  # hold out one in every 2000 lines as dev data.
+  zcat $text | cut -d ' ' -f2- | awk -v text_dir=$text_dir '{if(NR%2000 == 0) { print >text_dir"/dev.txt"; } else {print;}}' >$text_dir/librispeech.txt
```

Comment:
please use `gunzip -c`, not `zcat`, which I'm not sure if it would be installed everywhere under that name.

Reply:
sure, just changed it.